utils/src/server/rpc_module.rs

```diff
@@ -217,16 +223,18 @@ impl<Context> RpcContextModule<Context> {
 
 /// Used by the server to send data back to subscribers.
 #[derive(Clone)]
-pub struct SubscriptionSink {
+pub struct SubscriptionSink<Params> {
```

Comment:
Maybe adding a default here would prevent us from having turbofish `register_subscription::<()>`: 

```suggestion
pub struct SubscriptionSink<Params = ()> {
```

Not 100% that compiler will figure it out, but might be worth a try.

Reply:
Seems like it doesn't. I still like the suggestion of providing a default though, maybe the compiler will get smart enough one day.

Reply:
I think it will if we move the `register_subscription` to a trait with `P: Send + Sync + DeserializeOwned = ()`